lib/view_component/version.rb

```diff
@@ -3,7 +3,7 @@
 module ViewComponent
   module VERSION
     MAJOR = 2
-    MINOR = 0
+    MINOR = 1
```

Comment:
@joelhawksley would you prefer to do version bump and release in a separate PR? Just read the contributing guidelines...

Reply:
I would, if you don't mind removing this change 😄